src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/NetworkManager.cs

```diff
@@ -2,35 +2,36 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 using System.IO;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.SpectatorView
 {
     public abstract class NetworkManager<TService> : CommandRegistry<TService>, INetworkManager where TService : Singleton<TService>
     {
-        private float lastReceivedUpdate;
-
+#pragma warning disable 414 // The field is assigned but its value is never used
+        [Tooltip("Default prefab for creating an INetworkConnectionManager.")]
         [SerializeField]
-        protected TCPConnectionManager connectionManager = null;
+        private GameObject defaultConnectionManagerPrefab = null;
+        private GameObject connectionManagerGameObject = null;
+        protected INetworkConnectionManager connectionManager = null;
+#pragma warning restore 414
 
-        public bool AttemptReconnectWhenClient

```

Comment:
The reconnection logic across a few files seems to have been moved (removed?) but this is not mentioned in the PR description. What happened to it?

Reply:
hm, i felt like this reconnect logic was specific to the TCPConnectionManager compared to the NetworkManager. We could add this to the INetworkConnectionManager interface definition, but i don't know that all network connection managers would handle this functionality.

Right now if you want to disable attempting reconnections or enable, you need to update/create a new TCPConnectionManager prefab. I'll add this to the review description.

Reply:
Updated the description.